lib/src/emoji_picker_utils.dart

```diff
@@ -14,28 +15,29 @@ class EmojiPickerUtils {
 
   static final EmojiPickerUtils _singleton = EmojiPickerUtils._internal();
   final List<Emoji> _allAvailableEmojiEntities = [];
+  final _emojiRegExp = RegExp(r'(\p{So})', unicode: true);
 
   /// Returns list of recently used emoji from cache
   Future<List<RecentEmoji>> getRecentEmojis() async {
     return EmojiPickerInternalUtils().getRecentEmojis();
   }
 
   /// Search for related emoticons based on keywords
-  Future<List<Emoji>> searchEmoji(String keyword) async {
+  Future<List<Emoji>> searchEmoji(String keyword, List<CategoryEmoji> data,
```

Comment:
I wouldn't do that if it was completely up to me. I think, normally host app wants to search in the same set as it uses for display and with default value, it would be a very easy to overlook integration point. Requiring a value feels natural to me, but since you are responsible for the code base, I can add that. Let me know your final decision.

Reply:
I'm fine with that, just pointing out that this will be one of the breaking changes. Gonna need to mention that later in ChangeLog.

Reply:
It's not the only one, removing progress color from config would also be a breaking change, so I thought this might be alright.